website/docs/r/cloud_provider_snapshot_backup_policy.html.markdown

```diff
@@ -0,0 +1,118 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: cloud_provider_snapshot_backup_policy"
+sidebar_current: "docs-mongodbatlas-resource-cloud-provider-snapshot-backup-policy"
+description: |-
+    Provides an Cloud Provider Snapshot Backup Policy resource.
+---
+
+# mongodbatlas_cloud_provider_snapshot_backup_policy
+
+`mongodbatlas_cloud_provider_snapshot_backup_policy` provides a resource that enables you to view and modify the snapshot scheduling and retention settings for an Atlas cluster with Cloud Provider Snapshots enabled
+
```

Comment:
slight adjustment:
 `mongodbatlas_cloud_provider_snapshot_backup_policy` provides a resource that enables you to view and modify the snapshot schedule and retention settings for an Atlas cluster with Cloud Provider Snapshots enabled